schutzbot/mockbuild.sh

```diff
@@ -37,7 +37,7 @@ fi
 
 # Relative path of the repository – used for constructing both the local and
 # remote paths below, so that they're consistent.
-REPO_PATH=osbuild-composer/${DISTRO_VERSION}/${ARCH}/${COMMIT}
+REPO_PATH=osbuild-composer/${DISTRO_VERSION}/${ARCH}/${COMMIT}/nightly-${NIGHTLY:=false}
```

Comment:
Can we use ` REPO_PATH=osbuild-composer/${DISTRO_VERSION}/${ARCH}/${COMMIT}` as a default and ` REPO_PATH=osbuild-composer/nightly/${DISTRO_VERSION}/${ARCH}/${COMMIT}` for the nightlies? There might be references to these repos in other places and those would all break.

Reply:
Or `osbuild-composer-nightlies/${DIST....`